Frontend/components/orders/ordersTable.js

```diff
@@ -25,43 +25,40 @@ const OrdersTable = () => {
         
         <>
             <Title>Recent Orders</Title>
-            {!isSSR && orderStatus && orderDetailStatus === 'loading' && (
+            {!isSSR && orderStatus === 'loading' && (
                 <Typography>Loading data...</Typography>
             )}
-            {!isSSR && orderStatus && orderDetailStatus === 'error' && (
+            {!isSSR && orderStatus === 'error' && (
                 <Typography>Error fetching data</Typography>
             )}
 
-            {!isSSR && orderStatus && orderDetailStatus === "success" && (
+            {!isSSR && orderStatus === "success" && (
             <>
                 <Table size="small">
                     <TableHead>
                         <TableRow>
                             <TableCell>ID</TableCell>
                             <TableCell>Customer ID</TableCell>
                             <TableCell>Customer Name</TableCell>
-                            {/* <TableCell>Product</TableCell> */}
                             <TableCell>Date</TableCell>
                             <TableCell>Ship To</TableCell>
-                            <TableCell align="right">Quantity</TableCell>
                             <TableCell align="right">Invoice Amount</TableCell>
+                            <TableCell></TableCell>
                         </TableRow>
                     </TableHead>
                     <TableBody>
                         {orderData.map(order => (
                             <TableRow key={order["id"]}>
+                                <TableCell>{order["id"]}</TableCell>
                                 <TableCell>{order["customerId"]}</TableCell>
                                 <TableCell>{order["customerName"]}</TableCell>
-                                <TableCell>{order["orderDate"]}</TableCell>
+                                <TableCell>{Date(order["orderDate"]).toString()}</TableCell>
```

Comment:
I encountered this formatting problem before. There is a package called "moment" that we can use to format date correctly. If you want, you could try to implement it else I can take care of it :)

Reply:
I checked that before. Then i recognized that this package just format actual moment. Mayyybe i did wronggggggggg.   It will be nice when you could take care of it:) Thanks in advance 👍